tests/python_tests/test_continuous_batching.py

```diff
@@ -341,23 +423,23 @@ def test_preemption(params):
 
 
 # todo: Anastasiia Pnevskaya: fix the test because it is hanging according max_new_tokens = std::numeric_limits<std::size_t>::max()
-@pytest.mark.parametrize("dynamic_split_fuse", [True, False])
 @pytest.mark.precommit
+@pytest.mark.parametrize("dynamic_split_fuse", [True, False])
 @pytest.mark.skip(reason="Random sampling results are non deterministic due to: discrete_distribution impl depends on platform, model inference results may depend on CPU. Test passes on CI but fails locally.")
-def test_preemption_with_multinomial(dynamic_split_fuse):
+def test_preemption_with_multinomial(model_facebook_opt_125m: OVConvertedModelSchema, dynamic_split_fuse):
     generation_configs = multinomial_params.generation_config
     for config in generation_configs:
         config.max_new_tokens = 30
-    model_id : str = "facebook/opt-125m"
-    model, hf_tokenizer, models_path = download_and_convert_model(model_id)
 
     scheduler_config = dict_to_scheduler_config({"num_kv_blocks": 3, "dynamic_split_fuse": dynamic_split_fuse, "max_num_batched_tokens": 256, "max_num_seqs": 256})
-    generate_and_compare(model=models_path,
-                         pipeline_type=PipelineType.CONTINUOUS_BATCHING,
-                         prompts=multinomial_params.prompts,
-                         ref=multinomial_params.ref_texts,
-                         generation_config=generation_configs,
-                         scheduler_config=scheduler_config)
+    generate_and_compare(
+        model_schema=model_facebook_opt_125m,
+        pipeline_type=PipelineType.CONTINUOUS_BATCHING,
```

Comment:
Should the `llm_model` and `model_facebook_opt_125m` be parametrized with `pipeline_type`? That way `CONTINUOUS_BATCHING` won't be constructed multiple times. Or maybe just include all possible types into the schema and extract only needed

Reply:
Reminder